doc/api/crypto.md

```diff
@@ -5712,6 +5711,7 @@ See the [list of SSL OP Flags][] for details.
 [`util.promisify()`]: util.md#util_util_promisify_original
 [`verify.update()`]: #crypto_verify_update_data_inputencoding
 [`verify.verify()`]: #crypto_verify_verify_object_signature_signatureencoding
+[caveats when using strings as inputs to cryptographic APIs]: #crypto_using_strings_as_inputs_to_cryptographic_apis
```

Comment:
Shouldn't this go above line `5670`? Are we ordering this by the string inside `[]`?
```md
[`cipher.final()`]: #crypto_cipher_final_outputencoding
```

Reply:
Yes, the ordering is a "dumb" ASCII compare:
```js
'`'.charCodeAt() // 96
'c'.charCodeAt() // 99
'caveats when using strings as inputs to cryptographic APIs' > '`cipher.final()`' // true
```